helm/hpcc/templates/_warnings.tpl

```diff
@@ -0,0 +1,58 @@
+{{/*
+
+---  DO NOT EDIT THIS FILE ---
+
+##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2022 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+##############################################################################
+
+*/}}
+{{/*
+Generate a list of warnings
+Pass in dict with root with warnings
+*/}}
+{{ define "hpcc.getWarnings" }}
```

Comment:
minor: why was the whitespace chomping removed here (in commit 5) ?

All our helper functions have whitespace chomping here to avoid introducing new lines each time they are called

Reply:
Again, I removed when I was trying to get it to work.  I'll add back in.